roles/rsyslog/handlers/main.yaml

```diff
@@ -1,5 +1,5 @@
 ---
 - name: restart rsyslogd
-  service:
-    name: 'rsyslog'
-    state: 'restarted'
+  systemd:
```

Comment:
why this change?

Reply:
Well, at that time, I was just afraid that `service` could be deprecated some day. :p   Most likely, `service` would cover both non-systemd and systemd, wouldn't it?  If so, I'm going to revert this change.  Thanks!

Reply:
https://docs.ansible.com/ansible/latest/modules/service_module.html#status
"This module is guaranteed to have backward compatible interface changes going forward. [stableinterface]
This module is maintained by the Ansible Core Team. [core]"
So I don't think there is more risk of deprecation of `service` than for any other core Ansible feature. (In principle, any feature can become deprecated, what makes `service` special?)
Also, please follow examples of other roles. I believe we are using`service` at many other places. If it becomes deprecated, it will be a problem that we will solve globally for all roles. logging is not special and I don't see a reason to make it different.
Also please read the best practices here: https://github.com/oasis-roles/meta_standards#ansible-best-practices . Relevant part: "Use meta modules when possible. Instead of using the upstart and systemd modules, use the service module when at all possible. Similarly for package management, use package instead of yum or dnf or similar. This will allow our playbooks to run on the widest selection of operating systems possible without having to modify any more tasks than is necessary."
Also please try to avoid mixing unrelated changes in one PR - there already two unrelated changes in this PR and this change is not related to any of them.

Reply:
Thanks a lot for your comments, @pcahyna.

Indeed, it was wrong to include the unrelated changes.  I'm going to open separated pull requests instead of merging them into one.

Reply:
> Well, at that time, I was just afraid that `service` could be deprecated some day.

FYI, `service` will survive even the upcoming project restructuring: https://github.com/ansible-community/collection_migration/blob/6a42daee3d1a7443f64ebccff0994dcc29ecd439/scenarios/nwo/ansible.yml#L286